ethToElrond/v2/ethToElrondBridgeExecutor.go

```diff
@@ -200,6 +202,38 @@ func (executor *ethToElrondBridgeExecutor) PerformActionIDOnElrond(ctx context.C
 	return nil
 }
 
+// IsMaxRetriesReachedOnElrond checks if the retries on waiting were reached and increments the counter
+func (executor *ethToElrondBridgeExecutor) IsMaxRetriesReachedOnElrond() bool {
```

Comment:
we should rename this: Is.... will induce the thought that no internal state will be changed and only one value will be returned:
How about:
`ProcessMaxRetriesOnElrond` ?